.github/workflows/merlin.yml

```diff
@@ -10,7 +10,7 @@ on:
 env:
   ARTIFACT_RETENTION_DAYS: 7
   DOCKER_BUILDKIT: 1
-  GO_VERSION: 1.18
+  GO_VERSION: '1.20'
```

Comment:
Why the single quotes?

Reply:
It'll be truncated to 1.2 without it and the linting step will fail.

Reply:
Ahh ok.